### ISSUE TITLE
Incorrectly receive `statements are not aligned` error with `align` rule

### ISSUE THREAD

**Issue by unicron** [palantir/tslint#2640]
### Bug Report

- __TSLint version__: 5.1.0
- __TypeScript version__: 2.2.2
- __Running TSLint via__: (pick one) Node.js

#### TypeScript code being linted

```ts
import Backbone = require('backbone');
import Marionette = require('backbone.marionette');
import { LoggingService, LogLevel } from '../util/logging/LoggingService';

export class AitrItemView<T extends Backbone.Model> extends Marionette.ItemView<T> {
    public handleError(error: any, command: string) {
        error = error || 'Error processing command';
        let message = error.message || error.Message || error;
        if (command) {
            message = command + ' failed.\n    ' + message;
        }

        LoggingService.log(message, LogLevel.error, error);
    }
}
```

with `tslint.json` configuration:

```json
{
    "defaultSeverity": "error",
    "extends": [
        "tslint:latest"
    ],
    "jsRules": {},
    "rules": {
        "quotemark": [true, "single", "avoid-escape"],
        "max-line-length": [false],
        "prefer-const": false,
        "max-classes-per-file": [false],
        "ordered-imports": false,
        "object-literal-sort-keys": false,
        "comment-format": false,
        "only-arrow-functions": false,
        "member-ordering": false,
        "member-access": false,
        "no-bitwise": false,
        "trailing-comma": false,
        "object-literal-shorthand": false,
        "variable-name": [true, "ban-keywords", "check-format"],
        "no-var-requires": false,
        "no-consecutive-blank-lines": [true, 2],
        "one-line": [true, "check-catch", "check-finally", "check-else", "check-open-brace"],
        "interface-name": false,
        "object-literal-key-quotes": [true, "consistent"],
        "ban-types": false,
        "space-before-function-paren": false,
        "typedef": [true, "parameter", "arrow-parameter", "property-declaration", "member-variable-declaration", "object-destructuring", "array-destructuring"],
        "no-namespace": false,
        "prefer-for-of": false
    },
    "rulesDirectory": []
}
```

#### Actual behavior
running `npm run tslint` runs `tslint --project ./tslint.json` which produces the following errors:
ERROR: app/aitr/Aitr.ItemView.ts[2, 1]: statements are not aligned
ERROR: app/aitr/Aitr.ItemView.ts[3, 1]: statements are not aligned
ERROR: app/aitr/Aitr.ItemView.ts[5, 1]: statements are not aligned

If run with --fix option, the code is modified to insert a single space as the first character in front of all three lines, as below.  This is incorrect, as the statements are already aligned.

```ts
import Backbone = require('backbone');
 import Marionette = require('backbone.marionette');
 import { LoggingService, LogLevel } from '../util/logging/LoggingService';

 export class AitrItemView<T extends Backbone.Model> extends Marionette.ItemView<T> {
    public handleError(error: any, command: string) {
        error = error || 'Error processing command';
        let message = error.message || error.Message || error;
        if (command) {
            message = command + ' failed.\n    ' + message;
        }

        LoggingService.log(message, LogLevel.error, error);
    }
}
```

This error is repeated for MANY other files, but not for all files in my project.

#### Expected behavior
No errors generated from above code.

**Comment by ajafff** [comment]
This happens for files encoded in UTF8 with BOM, when the first statement is on the first line.
I recently fixed the same problem for `jsdoc-format`. I'm already working on a fix.

**Comment by unicron** [comment]
Thanks for getting this done quickly.
